test/test_abi_variants.py

```diff
@@ -184,9 +192,9 @@ def test_abi_none(tmp_path, capfd):
         project_dir,
         add_env={
             "CIBW_TEST_REQUIRES": "pytest",
-            "CIBW_TEST_COMMAND": "pytest {project}/test",
+            "CIBW_TEST_COMMAND": "python -m pytest {project}/test",
```

Comment:
It would be great to dig into why this change is needed, since it's indicative of a bug in the pyodide virtual environment code.

Reply:
Opened a Pyodide bug to track this https://github.com/pyodide/pyodide/issues/4802

Reply:
It seems that the issue only happens on macOS (either arm64 or x86_64). I tested on linux (aarch64/x86_64) and there were no issues.

Reply:
I added a commit that workaround the issue in our tests and a link to the issue in the docs as we're saying that macOS is not fully supported.